multivac/gather_job_data.py

```diff
@@ -162,6 +164,50 @@ def gather_data(self):
                 )
             self.gathered_data[job_id] = gathered_job_data
 
+    def put_to_db(self):
+        bucket = os.getenv('INFLUX_BUCKET')
+        org = os.getenv('INFLUX_ORG')
```

Comment:
We might have other buckets (like in S3) and orgs (like on GitHub) in this code. 

```suggestion
        influx_bucket = os.getenv('INFLUX_BUCKET')
        influx_org = os.getenv('INFLUX_ORG')
```